lib/Gedmo/Sortable/SortableListener.php

```diff
@@ -465,9 +465,9 @@ public function postFlush(EventArgs $args)
                                 // If the object implements Comparable interface we can use its compareTo method
                                 // Otherwise we fallback to normal object comparison
                                 if ($gr instanceof Comparable) {
-                                    $matches = $gr->compareTo($value);
+                                    $matches = 0 === $gr->compareTo($value);
                                 } else {
-                                    $matches = $gr == $value;
+                                    $matches = $gr === $value;
```

Comment:
We want to check for identical values here?
Looking at a previous condition, I think we already know that these values are not identical:
https://github.com/doctrine-extensions/DoctrineExtensions/blob/84b5768c37b989508ec06cc20c1d5ff554a10bc1/lib/Gedmo/Sortable/SortableListener.php#L463